### DISCUSSION TITLE
can i import the utils of the `@ariakit/react-core` ?

### DISCUSSION THREAD

**Discussion by BoneTM** [ariakit/ariakit#2404] [Questions] [answered]
any possible ?
![image](https://github.com/ariakit/ariakit/assets/48622532/cf3f7896-f612-4e4d-b311-23837f1c75f6)

**Comment by BoneTM** [comment, marked as answer]
nvm, i have import from 
![image](https://github.com/ariakit/ariakit/assets/48622532/8642c84a-dc57-4958-9c3a-5cd4f54e3b82)
by changing pnpm into npm

**Reply by diegohaz** [reply]
Stay advised that that API is not covered by semver. It will likely introduce breaking changes on patch and minor versions.

**Reply by BoneTM** [reply]
> Stay advised that that API is not covered by semver. It will likely introduce breaking changes on patch and minor versions.

thanks for reminding
